modules/sdk-coin-avaxp/test/unit/lib/transactionBuilder.ts

```diff
@@ -42,4 +44,59 @@ describe('AvaxP Transaction Builder', () => {
       );
     });
   });
+
+  describe('should json ', () => {
```

Comment:
nit: please explain what this test suite is doing

Reply:
done